vendor/assets/leaflet/leaflet.osm.js

```diff
@@ -64,6 +64,16 @@ L.OSM.HOT = L.OSM.TileLayer.extend({
   }
 });
 
+L.OSM.GPS = L.OSM.TileLayer.extend({
+  options: {
+    url: document.location.protocol === 'https:' ?
+      'https://{s}.gps-tile.openstreetmap.org/lines/{z}/{x}/{y}.png' :
+      'http://{s}.gps-tile.openstreetmap.org/lines/{z}/{x}/{y}.png',
+    maxZoom: 20,
+    subdomains: 'abc'
+  }
+});
+
```

Comment:
So I basically create a new pull request with https://github.com/jfirebaugh/leaflet-osm first and remove that part here then?